Makefile

```
@@ -123,12 +123,13 @@ ec/install:
 	$(@D) --version
 	@ echo "[$@]: Completed successfully!"
 
+install/pip: CUSTOM_VERSION_CMD ?= ''
 install/pip/%: PYTHON ?= python3
 install/pip/%: | guard/env/PYPI_PKG_NAME
 	@ echo "[$@]: Installing $*..."
 	$(PYTHON) -m pip install --user $(PYPI_PKG_NAME)
 	ln -sf ~/.local/bin/$* $(BIN_DIR)/$*
-	$* --version
+	$(if $(CUSTOM_VERSION_CMD),$(CUSTOM_VERSION_CMD),$* --version)
```

and with the above change, this could just be:

```suggestion
	$(PIP_PKG_VERSION_CMD)
```